cookbooks/aws-parallelcluster-install/resources/nvidia_dcgm/nvidia_dcgm_redhat8.rb

```diff
@@ -17,3 +17,9 @@
 end
 
 use 'partial/_nvidia_dcgm_common.rb'
+
+action :setup do
+  return unless node['cluster']['nvidia']['enabled'] == 'yes' || node['cluster']['nvidia']['enabled'] == true
+
+  action_install_package
+end
```

Comment:
The action cannot be shared since `_nvidia_dcgm_alinux2_centos7.rb ` has a slightly different condition and to directly show the condition of the setup action.